doc/api/domain.markdown

```diff
@@ -308,56 +269,55 @@ than crashing the program.
 
 * {Array}
 
-An array of timers and event emitters that have been explicitly added
-to the domain.
+Returns an array of timers and `EventEmitter` objects that have been explicitly
+bound to the domain.
 
 ### domain.add(emitter)
 
 * `emitter` {EventEmitter | Timer} emitter or timer to be added to the domain
 
-Explicitly adds an emitter to the domain.  If any event handlers called by
-the emitter throw an error, or if the emitter emits an `'error'` event, it
-will be routed to the domain's `'error'` event, just like with implicit
-binding.
+Explicitly binds an `EventEmitter` or timer to the domain.  If any event
+handlers called by the emitter throw an error, or if the `EventEmitter` emits
+an `'error'` event, the error will be routed to the `Domain` objects own
+`'error'` event.
 
-This also works with timers that are returned from [`setInterval()`][] and
-[`setTimeout()`][].  If their callback function throws, it will be caught by
-the domain 'error' handler.
+Timers created and returned by [`setInterval()`][] and [`setTimeout()`][] can
+be added to the `Domain`. If the timer callback function throws, the error will
+be passed to the `Domain` `'error'` handler.
 
-If the Timer or EventEmitter was already bound to a domain, it is removed
-from that one, and bound to this one instead.
+If the timer or `EventEmitter` was already bound to a `Domain`, it will be  
+removed from that one, and bound to this one instead.
 
 ### domain.remove(emitter)
 
 * `emitter` {EventEmitter | Timer} emitter or timer to be removed from the domain
 
-The opposite of [`domain.add(emitter)`][].  Removes domain handling from the
-specified emitter.
+Removes the specified `EventEmitter` or timer from this `Domain`.
 
 ### domain.bind(callback)
 
 * `callback` {Function} The callback function
 * return: {Function} The bound function
 
-The returned function will be a wrapper around the supplied callback
-function.  When the returned function is called, any errors that are
-thrown will be routed to the domain's `'error'` event.
+Wraps the supplied `callback` function in a new function that catches any
+errors thrown by the `callback` and routes those to the `Domain` objects
+`'error'` event.
+
+For example:
 
-#### Example
+    const domain = require('domain');
+    const myDomain = domain.create();
 
-    const d = domain.create();
+    const handler = myDomain.bind((err, data) => {
+      return cb(err, data ? JSON.parse(data) : null);
```

Comment:
This bound handler needs to be made *inside* readSomeFile so `cb` actually exists.